tests/integration-tests/conftest.py

```diff
@@ -602,6 +602,7 @@ def _config_renderer(config_file="pcluster.config.yaml", benchmarks=None, output
             raise FileNotFoundError(f"Cluster config file not found in the expected dir {config_file_path}")
         output_file_path = test_datadir / output_file if output_file else config_file_path
         default_values = _get_default_template_values(vpc_stack, request)
+        kwargs = inject_internal_storage_settings(**kwargs)
```

Comment:
Is it not possible to inject the internal storage settings the same way we render the other values in the template like in like 607?

Reply:
Line 607 is jinja's built in function.

Reply:
And line 608 is the one which replaces these default and keyword arguments into the template